packages/lesswrong/components/posts/PostsPage/PostsPageEventData.tsx

```diff
@@ -6,17 +6,22 @@ const styles = (theme: ThemeType): JssStyles => ({
     marginTop:theme.spacing.unit*3,
     ...theme.typography.postStyle,
     color: 'rgba(0,0,0,0.5)',
-  }
+  },
 })
 
 const PostsPageEventData = ({classes, post}: {
   classes: ClassesType,
   post: PostsBase,
 }) => {
-  const { location, contactInfo } = post
+  const { location, contactInfo, onlineEvent } = post
+  
+  const locationNode = onlineEvent ? (
+    <div className={classes.eventLocation}>Online Event</div>
+  ) : location && <div className={classes.eventLocation}> {location} </div>;
```

Comment:
Parentheses to make order of operation clearer
```suggestion
  ) : (location && <div className={classes.eventLocation}> {location} </div>);
```